<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSemester</class>
 <widget class="QWidget" name="AddSemester">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>139</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>356</width>
    <height>139</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>405</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Semester</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(6, 12, 10);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="DataInput">
     <item row="0" column="0">
      <widget class="QLabel" name="label_SemesterNumber">
       <property name="styleSheet">
        <string notr="true">font: 700 11pt &quot;Graduate&quot; ;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Semester Number</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_SemesterNumber">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(192, 191, 188);</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_StartDate">
       <property name="styleSheet">
        <string notr="true">font: 700 11pt &quot;Graduate&quot; ;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Start Date</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="dateEdit_StartDate">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(246, 245, 244);</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 245, 244);</string>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
